documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces-payload.adoc

```diff
@@ -325,7 +325,7 @@ e|https://admin-shell.io/aas/API/3/0/SubmodelRepositoryServiceSpecification/SSP-
 e|https://admin-shell.io/aas/API/3/0/SubmodelRepositoryServiceSpecification/SSP-002 |Indicates that the server implemented all details of the Submodel Service Repository Specification Read Profile in version 3.0.
 e|https://admin-shell.io/aas/API/3/0/SubmodelRepositoryServiceSpecification/SSP-003 |Indicates that the server implemented all details of the Submodel Service Repository Specification Read Profile in version 3.0.
 e|https://admin-shell.io/aas/API/3/0/SubmodelRepositoryServiceSpecification/SSP-004 |Indicates that the server implemented all details of the Submodel Service Repository Specification Template Profile in version 3.0.
-e|https://admin-shell.io/aas/API/3/0/ConceptDescriptionServiceSpecification/SSP-001 |Indicates that the server implemented all details of the Concept Description Service Specification Read Template Profile in version 3.0.
```

Comment:
wyh removed: we need both, no 3.0 and 3.1?

Reply:
This is also a slightly bigger topic. Do we want to list all the 3.0 and 3.1 profiles in this list? I guess so...

Reply:
Decision: Mirror fix also in the V3.0.4 for the ConceptDescriptionRepo... *but* also extend the list with all other `.../3/1/...` profile versions. In addition, change the Type for ServiceDescription/profiles to `Identifier` and write a sentence that the enum is only for IDTA-standardised profile name suggestions.